package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint src/* --ext .js",
     "prepare": "npm run build",
     "plop": "plop",
-    "test": "jest",
+    "test": "jest -w 1",
```

Comment:
As this is only an issue occurring in the ci I would add this parameter as part of the main.yaml eve pipeline file (https://github.com/scality/core-ui/blob/development/1.0/eve/main.yml#L29 that would became `npm run test -- -w 1` )instead in order to keep fast test execution in local dev env.

Reply:
Make sense!